index.html

```diff
@@ -19,17 +19,18 @@
         />
         <link rel="stylesheet" href="loginstyles.css" />
         <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://neo-ui.netlify.app/styles.css">
         
     </head>
     <body>
-        <nav class="navbar navbar-white bg-white">
-            <p>MyWebsite</p>
+        <nav class="navbar">
+            <p class="head">MyWebsite</p>
```

Comment:
Add an anchor tag here with your landing page link

Reply:
I have added an anchor tag in the login button.